File: engine/model/src/main/scala/net/server/channel/packet/party/PartySearchStartPacket.scala
package net.server.channel.packet.party

import net.server.MaplePacket

class PartySearchStartPacket( private var _min: Int,  private var _max: Int,  private var _members: Int,  private var _jobs: Int) extends MaplePacket {
     def min: Int = _min
     def max: Int = _max
     def members: Int = _members
     def jobs: Int = _jobs
}

File: engine/engine-model/src/main/java/net/server/channel/packet/party/PartySearchStartPacket.java
package net.server.channel.packet.party;

import net.server.MaplePacket;

public class PartySearchStartPacket implements MaplePacket {
   private final int min;

   private final int max;

   private final int members;

   private final int jobs;

   public PartySearchStartPacket(int min, int max, int members, int jobs) {
      this.min = min;
      this.max = max;
      this.members = members;
      this.jobs = jobs;
   }

   public int min() {
      return min;
   }

   public int max() {
      return max;
   }

   public int members() {
      return members;
   }

   public int jobs() {
      return jobs;
   }
}
